src/explore.sh

```diff
@@ -90,3 +98,18 @@ function explore_files_under_git()
 
   cmd_manager "$flag" "git grep -e \"$regex\" -nI $path"
 }
+
+function explore_files_using_grep()
+{
+  local regex="$1"
+  local path="$2"
+  local flag="$3"
+
+  # Silent by default
+  flag=${flag:-"SILENT"}
+
+  # If user only set regex value
+  path=${path:-"."}
+
+  cmd_manager "$flag" "grep -nrwI $path -e \"$regex\""
```

Comment:
We currently don use the `-w` flag in the "git grepping mode". So, for consistency, I think we should remove it here or add it in the git-grep search as well. Personally, I prefer not to use `-w`, since we wouldn't allow users to search for "non-words". And they can always resort to something like `"\bpattern\b"`, to simulate the same effect, without `-w`.

Reply:
I agree with the homogeneity for the feature as a whole. So, I will remove -w in my next version